### PR TITLE
Transaction class added. Methods for balance, deposit, withdrawal added.

### PR DESCRIPTION
**Pull request by mileseu**
Hi all, I've implemented methods for balance, deposit and withdraw. These are tracked by a new transaction class which will be added onto the Account class as an arraylist at a later stage.

As I mentioned in slack, my coding is shaky and my management of IntelliJ and Github is even worse! I wanted to post what I had first, as the fear of showing nothing for two weeks was getting to me! I hope it's ok to take a look at, and as a sort of start.

### CODE REVIEW

File: newbank/server/Transaction.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    public String getSummaryLine(){
+
+        if (this.amount >= 0){
+            return String.format("%s : £%.02f : %s", this.timestamp.toString(), this.amount, this.memo);
+        } else {
+            return String.format("%s : £(%.02f) : %s", this.timestamp.toString(), -this.amount, this.memo);
+        }
+
+    }
```
Comment (carolinamartes) [inline newbank/server/Transaction.java:41]: I really like this idea around transactions. If we call methods in Account that withdraw (currentBalance -= amount) and deposit that would be great 👍

**Comment by mileseu** [comment]
[- Users can withdraw money from an account](https://trello.com/c/AVGj3Ufh/29-users-can-withdraw-money-from-an-account)

**Comment by mileseu** [comment]
[- Users can deposit money into an account](https://trello.com/c/2MC7ExIc/28-users-can-deposit-money-into-an-account)

**Comment by mileseu** [comment]
[-Users can check their account balance](https://trello.com/c/k94djGUx/37-users-can-check-their-account-balance)

File: newbank/server/Transaction.java (hunk 1 above)
Comment (mileseu) [reply newbank/server/Transaction.java:41]: Thanks Carolina. I think we should continue with the logic you have used as I have written on your PR. We can close this with for now and use snippets instead.

**Comment by mileseu** [comment]
Replaced with #8 . Transfer class may be used from this in future build.

**Pull request closed without merging**